### PR TITLE
[FIX] 9.0 travis fix

### PR DESCRIPTION
**Pull request by manawi**

### CODE REVIEW

File: pos_debt_notebook/README.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Contributors
 
 Sponsors
 --------
-* `IT-Projects LLC <https://it-projects.info>`__
+* `IT-Projects LLC <https://it-projects.info>`___
```
Comment (yelizariev) [inline pos_debt_notebook/README.rst]: revert triple underscore

**Review by yelizariev — CHANGES_REQUESTED** [review]
```
  find . -type f -name '*.rst' | xargs sed -i 's/___/__/g'
```

**Comment by manawi** [comment]
@yelizariev ,done

**Comment by yelizariev** [comment]
:+1:

**Comment by yelizariev** [comment]
I'm going to merge this PR first https://github.com/it-projects-llc/pos-addons/pull/161

**Comment by yelizariev** [comment]
Fix conflicts: 
This branch has conflicts that must be resolved

**Comment by yelizariev** [comment]
fix this PR first
https://github.com/it-projects-llc/pos-addons/pull/155

**Comment by yelizariev** [comment]
This branch has conflicts that must be resolved

File: pos_debt_notebook/README.rst (hunk 1 above)
Comment (yelizariev) [inline pos_debt_notebook/README.rst]: triple underscore still present

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by manawi** [comment]
@yelizariev done

**Pull request merged**